docs_src/src/pages/documentation/api_reference/openapi.mdx

```diff
@@ -215,6 +215,40 @@ app.include_router(subrouter)
 
 </CodeGroup>
 
+## Other Specification Params
+
+We support all the params mentioned in the latest OpenAPI specifications (https://swagger.io/specification/). See an example of using request body below.
+
+<CodeGroup title="Request Body">
+
+```python {{ title: 'untyped' }}
+class CustomBody(TypedDict):
+    file: bytes
+    file_name: str
+
+
+@app.post("/sync/multipart-file")
+def sync_multipart_file(request, body=CustomBody):
+    files = request.files
+    file_names = files.keys()
+    return {"file_names": list(file_names)}
```

Comment:
@VishnuSanal , this was an example , we should improve the docs with a working example once the code is ready